sqflite/lib/src/sentry_sqflite.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:sqflite/sqflite.dart';
+
+import 'sentry_database.dart';
+
+Future<Database> openDatabase(
+  String path, {
+  int? version,
+  OnDatabaseConfigureFn? onConfigure,
+  OnDatabaseCreateFn? onCreate,
+  OnDatabaseVersionChangeFn? onUpgrade,
+  OnDatabaseVersionChangeFn? onDowngrade,
+  OnDatabaseOpenFn? onOpen,
+  bool readOnly = false,
+  bool singleInstance = true,
+}) {
+  Future<Database> openDatabase() async {
```

Comment:
Users would need to use our `openDatabase` in order to track APM for sqflite.
Or they can wrap directly with `SentryDatabase(Database)`